src/constants.ts

```diff
@@ -12,21 +8,13 @@ export const BLOCK_RATE_SECONDS = 13.14;
 
 export const TOKEN_DECIMALS = 9;
 
-interface IPoolGraphURLS {
-  [index: string]: string;
-}
-
-export const POOL_GRAPH_URLS: IPoolGraphURLS = {
-  4: "https://api.thegraph.com/subgraphs/name/pooltogether/rinkeby-v3_4_3",
-  1: "https://api.thegraph.com/subgraphs/name/pooltogether/pooltogether-v3_4_3",
+export const POOL_GRAPH_URLS: Partial<Record<NetworkId, string>> = {
+  [NetworkId.TESTNET_RINKEBY]: "https://api.thegraph.com/subgraphs/name/pooltogether/rinkeby-v3_4_3",
+  [NetworkId.MAINNET]: "https://api.thegraph.com/subgraphs/name/pooltogether/pooltogether-v3_4_3",
 };
 
-interface IAddresses {
-  [key: number]: { [key: string]: string };
-}
-
-export const addresses: IAddresses = {
-  4: {
+export const addresses: Record<NetworkId, Record<string, string>> = {
```

Comment:
if we just remove the typing ts can infer and autocomplete the addresses, but it will throw on some places where there are no address for the network being passed, don't know if it's worth making the change rn
![image](https://user-images.githubusercontent.com/6232729/147239822-f22f977c-ea50-42e5-8bd2-f8c1ef5a94e8.png)

Reply:
Agreed, if inference allows us to use autocomplete I think it's worth removing the explicit types.

Won't change it right now though, am still playing around to find possibly better ways of handling contracts and their addresses.